### PR TITLE
Minor syntax correction 

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by YYStreet** [comment]
Please skip the test_coverage function for now

File: test/dlc_tests/sanity/test_generate_coverage_doc.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from test.test_utils import TEST_COVERAGE_FILE
 
 
+@pytest.mark.skip()
```
Comment (saimidu) [inline test/dlc_tests/sanity/test_generate_coverage_doc.py:9]: Better to give reason for skip.

Comment (TusharKanekiDey) [reply test/dlc_tests/sanity/test_generate_coverage_doc.py:9]: Valid point.

Comment (YYStreet) [reply test/dlc_tests/sanity/test_generate_coverage_doc.py:11]:
```
@pytest.mark.skip(reason="REASON_TO_SKIP")
```
This pattern also works.

**Pull request merged**